README.md

```diff
@@ -33,7 +33,13 @@ Please send hiring@truss.works a link to a public git repository
 how to build and run it. Your code will be run on either macOS 10.15
 or Ubuntu 16.04 LTS, your choice.
 
-## The problem: CSV normalization
+## The problem: CSV or JSON data normalization
+
+You can decide between using the command line or browser-based code to
+complete the solution. Follow the instructions below based on which
+method you would like to use.
```

Comment:
I'd consider adding a sentence that emphasizes that the candidate should only pick one solution and that they should not do both.